### PR TITLE
PTTB-7 invitedUserId changed to id for InviteResource

### PR DESCRIPTION
**Pull request by Rznjoc**
wrong id was passed.

### CODE REVIEW

File: app/Http/Resources/InviteResource.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ public function toArray($request)
             'role_id'=> $this->role_id,
             'token'=> $this->token,
             'tokenExpires'=> $this->tokenExpires,
-            'invitedUserId'=> $this->invitedUserId,
+            'invitedUserId'=> $this->id,
```
Comment (Prabeg29) [inline app/Http/Resources/InviteResource.php:25]: @Rznjoc `invitedUserId` No need to be this verbose, just have the key as `id`. For the value role_id, now that needs to be verbose. How would I know, what role the invited user would have, just from this?

**Comment by Rznjoc** [comment]
Dai maybe its because only Admin can send invites to other users? Correct
me if I'm wrong.

**Review by Rznjoc — COMMENTED** [review]
please rewiew this.

**Comment by Prabeg29** [comment]
@Rznjoc The `InvitedResource` you have created, is being used by `[InviteController::class, 'listInvitedUsers"']` where  the service `InviteService::invitedList()` returns a collection of model `InviteToken` that has relation to `role`. Showing role information won't be such hard.

So, admin or whatever user I am, I get the list of users' information to whom I had sent invitations. Now think from the perspective of FE dev, the FE might show these records in some table where the `role` column will have `role_id`. So, do I hit roles API / check the database to see what role will the user have? Is that a good UX?

**Review by Prabeg29 — APPROVED** [review]

**Pull request merged**